### PR TITLE
Update email language #656, #661.

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: config/locales/requests.en.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ en:
       brief_msg: "Item offsite at Forrestal Annex. Request for pick-up."
       # temporary changes issue 438
       #email_conf_msg: "You will be notified via email when your item is ready for pick-up. Most items are ready in 1 to 2 business days."
-      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 12-2."
+      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 11am - 3pm, Monday - Friday.."
```
Comment (carolyncole) [inline config/locales/requests.en.yml:46]: @kevinreiss this has an extra period at the end.

File: config/locales/requests.en.yml [hunk 2]
```diff
@@ -52,22 +52,22 @@ en:
       brief_msg: "Item offsite at Fine Annex. Request for delivery in 1-2 business days."
       # temporary changes issue 438
       #email_conf_msg: "You will be notified via email when your item is ready for pick-up. Most items are ready in 1 to 2 business days."
-      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 12-2."
+      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 11am - 3pm, Monday - Friday.."
```
Comment (carolyncole) [inline config/locales/requests.en.yml:55]: This also has the extra period

File: config/locales/requests.en.yml [hunk 3]
```diff
@@ -52,22 +52,22 @@ en:
...
     ppl:
       email: "ppllib@princeton.edu"
       email_subject: "Harold P. Furth Plasma Physics Library Request"
       brief_msg: "Pageable item at Plasma Physics Library. Request for delivery in 1-2 business days."
       # temporary changes issue 438
       #email_conf_msg: "You will be notified via email when your item is ready for pick-up. Most items are ready in 1 to 2 business days."
-      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 12-2."
+      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 11am - 3pm, Monday - Friday.."
```
Comment (carolyncole) [inline config/locales/requests.en.yml:62]: This also has the additional period.  Also I am assuming when Lewis reopens the pickup will be in PPL not Firestone?

File: config/locales/requests.en.yml [hunk 4]
```diff
@@ -52,22 +52,22 @@ en:
...
       #brief_msg: "Pageable item at Lewis Library, will be delivered to Lewis Library Service desk on first floor."
       brief_msg: "Pageable item at Lewis Library. Request for delivery in 1-2 business days."
       #email_conf_msg: "Items requested before 3:00 pm (weekdays and weekends) will be available for pick-up, on the same day, at the Lewis Library Service desk on the first floor of the library. Items requested after 3:00 pm will be available the next day."
-      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 12-2."
+      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 11am - 3pm, Monday - Friday.."
```
Comment (carolyncole) [inline config/locales/requests.en.yml:70]: This also has the additional period.  Also I am assuming when Lewis reopens the pickup will be in Lewis not Firestone?

File: app/views/requests/request_mailer/_request_header_patron.html.erb [hunk 5]
```diff
@@ -13,7 +13,7 @@
       <table border="0" cellpadding="0" cellspacing="0" width="100%">
         <tr><td class="right-aligned-desktop" style="text-align:right;">
           <div style="font-size:25px;font-weight:bold;display:block;line-height:1.2;"><%= (DateTime.now + 7).strftime("%m/%d/%Y") %></div>
-          <span style="font-size:12px;display:block;">Hold Expiration Date</span>
+          <span style="font-size:12px;display:block;">Pick-up By:</span>
```
Comment (axamei) [inline app/views/requests/request_mailer/_request_header_patron.html.erb:16]: @kevinreiss I would get rid of the colon.

File: config/locales/requests.en.yml [hunk 6]
```diff
@@ -52,22 +52,22 @@ en:
...
     ppl:
       email: "ppllib@princeton.edu"
       email_subject: "Harold P. Furth Plasma Physics Library Request"
       brief_msg: "Pageable item at Plasma Physics Library. Request for delivery in 1-2 business days."
       # temporary changes issue 438
       #email_conf_msg: "You will be notified via email when your item is ready for pick-up. Most items are ready in 1 to 2 business days."
-      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 12-2."
+      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in PPL Library between the hours of 11am - 3pm, Monday - Friday.."
```
Comment (carolyncole) [inline config/locales/requests.en.yml:62]: @kevinreiss sorry!  Still has extra period...

File: config/locales/requests.en.yml [hunk 7]
```diff
@@ -52,22 +52,22 @@ en:
...
       #brief_msg: "Pageable item at Lewis Library, will be delivered to Lewis Library Service desk on first floor."
       brief_msg: "Pageable item at Lewis Library. Request for delivery in 1-2 business days."
       #email_conf_msg: "Items requested before 3:00 pm (weekdays and weekends) will be available for pick-up, on the same day, at the Lewis Library Service desk on the first floor of the library. Items requested after 3:00 pm will be available the next day."
-      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Firestone Lobby between the hours of 12-2."
+      email_conf_msg: "Your request to pick this item up has been received. We will process the requests as soon as possible, but given the demand and the current circumstances some delay may occur. You will receive an email the day prior telling you the item will be available for pick-up in Lewis Library Lobby between the hours of 11am - 3pm, Monday - Friday.."
```
Comment (carolyncole) [inline config/locales/requests.en.yml:70]: Here too...

**Pull request merged**